respa_admin/templates/respa_admin/resources/form/_general_info.html

```diff
@@ -34,4 +34,10 @@ <h4>{% trans "Contact person" %}</h4>
     <div class="external-calendar">
         <h4>{% trans "External calendar" %}</h4>
         {% include "respa_admin/forms/_input.html" with field=form.external_reservation_url %}
-</div>
+    </div>
+    <div class="external_reservation_text">
+        <h4>{% trans "External reservation free text" %}</h4>
```

Comment:
This text doesn't really make sense it's two statements without any context, I would at least have a comma there "External reservation, free text" or even better would be something like "note from external reservation". Can you either change it or come up with something else that fits in this context.